pnpm-lock.yaml

```diff
@@ -1818,130 +1818,6 @@ importers:
         specifier: ^5.8.0
         version: 5.10.0
 
-  examples/benchmarks/bubblebench/shared-tree-flex-tree:
```

Comment:
Nit: These changes seem unrelated.

Reply:
It looks like they are from a PR I did the other day, where I removed the shared-tree-flex-tree package. So this is probably good to check in. Is there something I need to do besides run `pnpm i` in order to regenerate the pnpm-lock? I thought that running `pnpm i` would have done this for me, but I'm pretty sure I ran `pnpm i` and it did not.

Reply:
If you want the lock file to update, you need to run `pnpm i --no-frozen-lockfile` now. That said if doing so would make a change, I'd expect pnpm i to error and tell you that,. I guess maybe there is some subset of changes which would update, but doesn't error?